powerstore/provider_test.go

```diff
@@ -0,0 +1,65 @@
+package powerstore
+
+import (
+	"context"
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-framework/providerserver"
+	"github.com/hashicorp/terraform-plugin-framework/tfsdk"
+	"github.com/hashicorp/terraform-plugin-go/tfprotov6"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"os"
+	"testing"
+)
+
+// var testAccProviders map[string]func() tfsdk.Provider
+var testProvider tfsdk.Provider
+var testProviderFactory map[string]func() (tfprotov6.ProviderServer, error)
+var endpoint = os.Getenv("ENDPOINT")
```

Comment:
The environment variable names look generic. Can we have specific to PowerStore e.g. POWERSTORE_ENDPOINT?

Reply:
updated